src/components/InformationPanes.js

```diff
@@ -31,17 +33,18 @@ function debugBtn(onClick, type, className, tooltip) {
   );
 }
 
-const RightSidebar = React.createClass({
+const InformationPanes = React.createClass({
```

Comment:
Hmm, so the more i think about it the more I wonder if `InformationPanes` is too vague.

My beef with information is largely that it could be applied to anything :)

One thought is that we could refer to the left and right sidebars as primary and secondary panes. At the moment, the left sidebar is just sources, but if I were to bet, it would get other items over time.